enatega-multivendor-admin/src/components/Food/Food.jsx

```diff
@@ -210,6 +210,12 @@ function Food(props) {
           ? event.target.value.toUpperCase()
           : event.target.value
       variationSetter([...variations])
+    } else if (type === 'discounted') {
+      // Enforce non-negative discounted price
+      const newValue = Math.max(0, parseFloat(event.target.value))
+      variations[index][type] = newValue
+      variations[index][type + 'Error'] = newValue < 0 // Update error based on new value
```

Comment:
Are you using this error anywhere ? because it gets stored inside variations array

Reply:
yes actually it checks that the discounted text field doesnot have any negative value

Reply:
yes, you are right im not using it anywhere